plugins/hypervisors/baremetal/src/main/java/com/cloud/baremetal/manager/BareMetalDiscoverer.java

```diff
@@ -205,7 +205,7 @@ public boolean stop() {
             zone.setDhcpProvider(Network.Provider.ExternalDhcpServer.getName());
             _dcDao.update(zone.getId(), zone);
 
-            s_logger.debug(String.format("Discover Bare Metal host successfully(ip=%1$s, username=%2$s, password=%3%s," +
+            s_logger.debug(String.format("Discover Bare Metal host successfully(ip=%1$s, username=%2$s, password=%3$s," +
                     "cpuNum=%4$s, cpuCapacity-%5$s, memCapacity=%6$s)", ipmiIp, username, "******", cpuNum, cpuCapacity, memCapacity));
```

Comment:
WIthout the fix it will print something as:
```
Discover Bare Metal host successfully(ip=ipmiIp, username=username, password=  %s,cpuNum=cpuNum, cpuCapacity-cpuCapacity, memCapacity=memCapacity)
```

WIth the fix:
```
Discover Bare Metal host successfully(ip=ipmiIp, username=username, password=******,cpuNum=cpuNum, cpuCapacity-cpuCapacity, memCapacity=memCapacity)
```